client/src/utils/filterHelper.ts

```diff
@@ -1,12 +1,12 @@
 
-export default function test(filter = '', ...values) {
+export default function test(filter = '', ...values: string[]) {
     const value = filter.toLowerCase();
     return values
         .filter(x => !!x)
         .some(x => x.toLowerCase().includes(value));
 }
 
-export function filterByOwner(items, owner) {
+export function filterByOwner(items: any[], owner: {[key: string]: any}) {
```

Comment:
I'd like to avoid using `any` and `{[key: string]: any}` types here. At some point we'll want to build out actual types for the k8s apis. Could we use `TODO` types here for now so it's clear that these are just temporary placeholders?

Reply:
Sure, I will use TODO to replace all these. I found that these may be structured object. We should be able to define them and reuse them later.